_data-prepper/pipelines/configuration/sinks/s3.md

```diff
@@ -131,14 +141,25 @@ Option | Required | Type | Description
 ### parquet codec
 
 The `parquet` codec writes events into a Parquet file.
-You must set the `buffer_type` to `multipart` when using Parquet.
+You must set the `buffer_type` to `in_memory` when using Parquet.
+
+The Parquet codec writes data using the Avro schema.
+Because Parquet requires an Avro schema, you may either define the schema yourself, or Data Prepper will automatically generate a schema.
+However, we generally recommend that you define your own schema so that it can best meet your needs.
+
+We recommend that you make your Avro fields use a null [union](https://avro.apache.org/docs/current/specification/#unions) unless you can be completely certain that the data will be present.
+Any events which do not have values for types which do not have a null union will fail to write to the destination.
```

Comment:
Do we need to recreate this since this same text already exists in Avro codec section? We could just link back to the Avro rules your outlined in the second sentence.